### DISCUSSION TITLE
Do not clear tuples in playground when there are problems with request

### DISCUSSION THREAD

**Discussion by luwenhegithub** [openfga/community#183] [Polls]
当使用Playground保存Tuples时，如果报错了，表单域里的内容会被清空。

希望这时候不会清空表单域的内容，以便更快捷的更改内容重新提交。

**Comment by adriantam** [comment]
Translation: 
If there are problems adding tuples in the playground, the tuples will be cleared. Keeping these tuples allow better user experience.

**Comment by adriantam** [comment]
@luwenhegithub : If possible, can you edit the category to be Idea?

如果可以的话, 能否将类别修改为Idea？谢谢!
